db/primary_migrate/20230518221444_add_postgis_extension_to_database.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostgisExtensionToDatabase < ActiveRecord::Migration[7.0]
+  def change
+    enable_extension 'postgis'
```

Comment:
wanted to double check, but it looks like this does do "IF NOT EXISTS":
https://github.com/rails/rails/blob/v7.0.5/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb#L433-L437

Reply:
Out of curiosity / learning, what drove you to check that? A possible error if it's already installed and we try to re-install it?

Reply:
Yep, exactly. If for some reason the extension was already created, a "CREATE EXTENSION" without "IF NOT EXISTS" would raise an error.